src/main/java/com/kruize/service/RecommendationsService.java

```diff
@@ -153,11 +160,64 @@ private JsonObject getResourceJson(ApplicationRecommendationsImpl applicationRec
 
             resourcesJson.add("requests", resourceRequestsJson);
             resourcesJson.add("limits", resourceLimitsJson);
+
+            if (applicationRecommendations.getRuntime(application) != null) {
+                try {
+                    JsonObject runtimeRecommendationJson = getRuntimeOptions(applicationRecommendations, application);
+                    resourcesJson.add("runtime_recommendations", runtimeRecommendationJson);
+                } catch (NoSuchApplicationException | NullPointerException ignored) { }
+            }
+
             return resourcesJson;
         }
 
         LOGGER.info("Application {} is no longer running and has no recommendations generated earlier", application);
         LOGGER.info("Not returning any recommendations");
         return null;
     }
+
+    private JsonObject getRuntimeOptions(ApplicationRecommendationsImpl applicationRecommendations, String application) throws NoSuchApplicationException, NullPointerException
+    {
+        if (applicationRecommendations.getRuntime(application).equals("java"))
+        {
+            return getJavaOptions(applicationRecommendations, application);
+        }
+        return null;
+    }
+
+    private JsonObject getJavaOptions(ApplicationRecommendationsImpl applicationRecommendations, String application) throws NoSuchApplicationException, NullPointerException
+    {
+        DecimalFormat precisionTwo = new DecimalFormat("#.##");
+        precisionTwo.setRoundingMode(RoundingMode.CEILING);
+
+        String labelName = applicationRecommendations.applicationMap.get(application)
+                .get(0).getLabelName();
+
+        JavaRecommendations javaRecommendations = JavaApplicationMetricsImpl.javaApplicationInfoMap
+                .get(labelName)
+                .getJavaRecommendations();
+
+        double heapRecommendations = javaRecommendations.getHeapRecommendation();
+
+        String gcPolicyRecommendation = javaRecommendations.getGcPolicy();
+
+        String percentage = precisionTwo.format((MathUtil.bytesToMB(heapRecommendations * 100)
+                /  applicationRecommendations.getRssLimits(application)));
```

Comment:
Isnt the getRssLimits in bytes as well and needs to be converted to MB ? If not, I would suggest that we need to be consistent and have all values sored either in MB or in bytes.

Reply:
Right now all the instances have the values in bytes, and all the applications have metrics in MB, so getRssLimits will be in MB.

Should I change all the instance recommendations, including rssRequests and rssLimits and the java recommendations to MB?